### PR TITLE
needed session token

### PR DESCRIPTION
**Pull request by benjaminleighton**

### CODE REVIEW

**Review by shaneseaton — COMMENTED** [review]
After a quick chat we might pull this env variable out into its own docker-compose file so it doesn't require the whole rebuild chain to depend on this var (its really only needed for a bespoke run of the RR<>CC generation).

After creating the new docker-compose files, the pattern in other directories/creators is to create a specific shell script per creation step. So something like `createCCtoRR.sh` and `createMBtoCC.sh`. If we can follow that, great, otherwise no biggie. See `docker/linksets/gnaf2geofab/create201605toCC.sh` as an example

**Comment by benjaminleighton** [comment]
It seems that the problem was that, by default, the bot-loci-uploader user didn't have sufficient permissions on the S3 bucket to actually perform any loci-asset s3 download operations after s3 bucket restrictions where added. I've excluded the bot-loci-uploader from those permissions and I'm now able to pass through just LOCI_S3_ACCESS_KEY_ID and LOCI_S3_SECRET_ACCESS_KEY and things seem to be working. However as discussed as part of solving this I've also adjusted the code to separate out rr>cc from mb>cc and followed the "creat***.sh" script pattern that wraps specifying docker-compose -f files. Testing is ongoing but loci-infrastructure has also been adapted to pass bot-loci-uploader keys rather than just using AWS_* keys and this appears to be working so far.

**Comment by benjaminleighton** [comment]
I've completed a loci infrastructure test and this is working. @shaneseaton @ashleysommer for re-review

File: docker/linksets/asgs2geofab/docker-compose.base.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+version: "3.4"
+
+services:
+  linksets:
+    build:
+      context: .
+      target: python-specific
+    env_file:
+      - ../../../common/common.env
+      - .env
```
Comment (shaneseaton) [inline docker/linksets/asgs2geofab/docker-compose.base.yml:10]: Not sure if .env files are the way to go here, or simply put ENV values directly into the docker-compose.yaml files? Not a problem, just a thought.

File: docker/linksets/asgs2geofab/start.sh [hunk 2]
```diff
@@ -3,4 +3,4 @@ set -e
 
 source ../../../common/common.sh
 
-docker-compose up -d --build --force-recreate
+docker-compose up -f docker-compose.base.yml -d --build --force-recreate
```
Comment (shaneseaton) [inline docker/linksets/asgs2geofab/start.sh:6]: Unsure what running base, on its own would achieve. Should this script be removed?

**Review by shaneseaton — APPROVED** [review]
Happy with the docker/workflow integration, and will leave other code stuff to someone who knows better.

However, I am concerned that there are copies of the headers in here from the repo's. This is a major problem and can only lead to bad things (diverging copies). We need to sort out where the 'official' version of the headers lives so we are not maintaining multiple copies. 

Note: I am not blameless here either, I have a copy of the headers in my linkset creators too. Need to resolve, and will make a github issue reflecting this.

**Pull request merged**